source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift

```diff
@@ -180,6 +180,10 @@ class ACRChoiceButton: NSView, NSTextFieldDelegate, InputHandlingViewProtocol {
         errorDelegate?.inputHandlingViewShouldShowError(self)
     }
     
+    func setShowError(_ value: Bool) {
```

Comment:
use of bool value if we always pass true?

Reply:
Yes, I had thought that. but If we will need to use it once further in a situation where its value can be both then meaning not met. believe, we ought to make it a set property.

Reply:
makes sense!